crates/ignore/src/gitignore.rs

```diff
@@ -772,6 +772,21 @@ mod tests {
         assert!(super::parse_excludes_file(&data).is_none());
     }
 
+    #[test]
+    fn parse_excludes_file4() {
+        let data = bytes("[core]\nexcludesFile = \"~/foo/bar\"");
+        let got = super::parse_excludes_file(&data);
+        assert_eq!(
+            path_string(got.unwrap()),
+            super::expand_tilde("~/foo/bar")
+        );
+    }
+    #[test]
```

Comment:
And also, code should follow the style of the surrounding code. In this case, there should be a blank
line separating test functions.

(You don't need to do this now. I've done it for you and this PR will get rolled up into another bigger one.)